cpp/src/bitmask/null_mask.cu

```diff
@@ -222,6 +282,42 @@ cudf::size_type count_unset_bits(bitmask_type const *bitmask, size_type start,
   return (num_bits - detail::count_set_bits(bitmask, start, stop, stream));
 }
 
+// Create a bitmask from a vector of column views
+void concatenate_masks(std::vector<column_view> const &views,
+    bitmask_type * dest_mask,
+    cudaStream_t stream) {
+  using CDViewPtr =
+    std::unique_ptr<column_device_view,
+                    std::function<void(column_device_view*)>>;
+  std::vector<CDViewPtr> cols;
+  thrust::host_vector<column_device_view> deviceViews;
+
+  thrust::host_vector<size_type> view_offsets(1, 0);
+  for (auto &v : views) {
+    cols.emplace_back(column_device_view::create(v, stream));
+    deviceViews.push_back(*(cols.back()));
+    view_offsets.push_back(v.size());
+  }
+  thrust::inclusive_scan(thrust::host,
+      view_offsets.begin(), view_offsets.end(),
+      view_offsets.begin());
+
+  rmm::device_vector<column_device_view> dViews = deviceViews;
```

Comment:
Use copy constructor rather than assignment?

```suggestion
  rmm::device_vector<column_device_view> d_views{device_views};
```

Reply:
If this is a style issue then let me know. In the general case if you write
```
A a = a1;
```
the copy constructor `A(a1)` is called instead of default constructor and an assignment operator.

Reply:
That's true. I just prefer functional initialization to assignment. It's a matter of preference, except for the case of the GCC compiler bug that @trevorsm7 ran into today.

Reply:
Fixed